README.md

```diff
@@ -141,6 +169,32 @@ NB: If the `destroy-microbosh` pipeline is run without first cleaning up
 CloudFoundry, it will be necessary to manually clean up the CloudFoundry
 deployment.
 
+### Connecting to your environment
```

Comment:
This is a really good section to include. It can be easy to forget that the end goal is making Cloud Foundry work!

It might be worth renaming the title though to something like "Connecting to Cloud Foundry" or "Connecting to Cloud Foundry in your environment".